internal/pkg/manifest/workload.go

```diff
@@ -23,7 +23,7 @@ const (
 )
 
 var (
-	errUnmarshalBuildOpts = errors.New("can't unmarshal build field into string or compose-style map")
+	errUnmarshalBuildOpts = errors.New("unmarshal build field into string or compose-style map")
 	errUnmarshalCountOpts = errors.New(`unmarshal "count" field to an integer or autoscaling configuration`)
```

Comment:
I think since it is the root error we should probably keep the `can't`. Otherwise it doesn't sound like anything wrong happened with "unmarshal build field into string or compose-style map".